clients/src/test/java/org/apache/kafka/clients/consumer/internals/FetcherTest.java

```diff
@@ -593,6 +593,40 @@ public void testFetchOffsetOutOfRangeException() {
     }
 
     @Test
+    public void testFetchPositionAfterOffsetOutOfRangeException() {
```

Comment:
Can we add some simple comments. So readers can understand the test more easily?